oci-repository-plugin/README.md

```diff
@@ -0,0 +1,151 @@
+# Build and install the plugin
```

Comment:
This duplicates parent README?

Reply:
Looks like it slipped through the cracks, removed the duplicated README with the old references to ES.